tutorials/slack-slash-command-app-engine-java.md

```diff
@@ -87,7 +87,7 @@ page](https://api.slack.com/apps) and click **Create new app**.
 ### Install the command to your team
```

Comment:
Should the heading also be changed to "workspace"?

Reply:
Sure, I guess that makes sense. Will update the PR.